apps/src/sites/code.org/pages/teacher-dashboard/index.js

```diff
@@ -1,18 +1,16 @@
 /**
- * Entry point for projects/section_projects.js bundle
+ * Entry point for teacher-dashboard/index.js bundle
  */
 
+/* globals window */
```

Comment:
surprised `window` isnt an implied global. the linter complains if you dont have this line? i guess maybe it's good in that it makes you be explicit about which files use window.